Adyen/UI/View Controllers/SearchViewController/SearchViewController.swift

```diff
@@ -0,0 +1,168 @@
+//
+// Copyright (c) 2023 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+@_spi(AdyenInternal)
+public protocol SearchViewControllerEmptyView: UIView {
```

Comment:
`SearchResultEmptyView` or `SearchResultsEmptyView` to indicate this is a result view and let's add documentation above

Reply:
Fixed with a8e8581345f4712fcf9fc0b1bf84449432edfcf5